edge_computing/policygentemplates_for_ztp/ztp-configuring-managed-clusters-policies.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ztp-configuring-managed-clusters-policies"]
+= Configuring managed clusters with policies and PolicyGenTemplate resources
+include::_attributes/common-attributes.adoc[]
+:context: ztp-configuring-managed-clusters-policies
+:policy-gen-cr: PolicyGenTemplate
+:policy-prefix:
+:argocd-folder: out/argocd/example/policygentemplates
```

Comment:
Should these custom attributes be closed at the end of this assembly?

Reply:
Yes indeed. Updated.